cmd/cosign/cli/flags.go

```diff
@@ -32,3 +40,19 @@ func nOf(args ...interface{}) int {
 	}
 	return n
 }
+
+type RegistryOpts struct {
+	AllowInsecure bool
+}
+
+func (co *RegistryOpts) GetRegistryClientOpts(ctx context.Context) []remote.Option {
+	opts := DefaultRegistryClientOpts(ctx)
```

Comment:
I wonder whether it's worth getting rid of this helper?  Just inline it so that folks use the appropriate new one?

Reply:
It _would_ gently nudge people in the right direction...